dev-local/msls/login_form.go

```diff
@@ -1,12 +1,80 @@
+<h2>Simulated Authentication no PHI or PII</h2>
+<h2>Use Only Synthetic Beneficiary Info</h2>
+<h2>Enter values to be returned by the SLS user_info endpoint below:</h2>
+</p>
+<br>
+<br>
+<form method="post" action="/login">
+<div>
+    <h3>Enter User Name:</h3>
+    <label>SUB(username): </label> <input type="text" name="username" size=50></input>
+    <br>
+    <h3>Enter Identity:</h3>
+    <label>BENE Id, HICN or HICN HASH, MBI or MBI HASH Accepted</label> <input type="text" name="usr_identity" size=64></input>
+    <br>
+    <br>
+    <p><b>Select Identity Type:</b></p>
+    <input type="radio" id="is_hicn" name="beneficiary_identity" value="H" checked="checked">HICN</input>
+    <br>
+    <input type="radio" id="is_hicn_hash" name="beneficiary_identity" value="HH" disabled>HICN Hash</input>
+    <br>
+    <input type="radio" id="is_mbi" name="beneficiary_identity" value="M">MBI</input>
+    <br>
+    <input type="radio" id="is_mbi_hash" name="beneficiary_identity" value="MH" disabled>MBI Hash</input>
+    <br>
+    <input type="radio" id="is_bene_id" name="beneficiary_identity" value="S" disabled>BENE ID</input>
+    <br>
```

Comment:
The SLS `user_info()` end point returns the following values only. Here is an example with fake values:

{'sub': '39ac983d-9d5a-872b-af8a-eac3984a939d', 'name': 'NAME', 'given_name': 'GIVEN_NAME', 'family_name': 'FAMILY_NAME', 'email': 'EMAIL', 'hicn': '1000057704', 'mbi': '4S10A00AA00'}

This inputs form should just include those values and should not have the radio button choices. We are wanting to test BB2 with both mbi and hicn values returned.

The mbi and hicn values are the NOT hashed ID's. SLS doesn't return any hashed values. These get hashed prior to BB2 performing the FHIR_ID/BENE_ID lookup.

All of the values input in this form should be passed thru to the userinfo return for MSLS, and should just include the same values as the real SLS end point return in order to simulate it correctly.

Reply:
identity_type removed